src/functions/Invoke-AzOpsPush.ps1

```diff
@@ -77,6 +77,7 @@
             if ($fileItem.Name.EndsWith('.parameters.json')) {
                 $result.TemplateParameterFilePath = $fileItem.FullName
                 $deploymentName = $fileItem.Name -replace (Get-PSFConfigValue -FullName 'AzOps.Core.TemplateParameterFileSuffix'), ''
+                $deploymentName = $deploymentName -replace ' ', '_'
```

Comment:
For consistency (see line 144), suggest that we keep this on one line here as well.
```suggestion
                $deploymentName = $fileItem.Name -replace (Get-PSFConfigValue -FullName 'AzOps.Core.TemplateParameterFileSuffix'), '' -replace ' ', '_'
```